apps/ui/src/components/pages/ErrorPage.tsx

```diff
@@ -26,7 +26,7 @@ const { Content } = Layout;
 
 type ErrorProps = {
 	loading: boolean;
-	error?: ServerError;
+	error: ServerError | null;
```

Comment:
more info on the comment further down. this works better if you seperate out the components.

What's interesting here is why would an `Error` component ever not have an error?
Only example I can think of is something like a default case, but even then I think `SeverError` should cover this. 
eg. maybe a union of a different errors like `type ServerError = FetchError | DefaultError`
This makes intent crystal clear, and a developer doesn't have to jump into the component to figure out why a null might be passed